_config.yml

```diff
@@ -15,7 +15,7 @@
 # in the templates via {{ site.myvariable }}.
 ---
 title: "Galaxy Training!"
-email: your-email@domain.com
+email: galaxytrainingnetwork@gmail.com
```

Comment:
off topic: would you like `training@usegalaxy.eu` to go anywhere special?

Reply:
GTN mailing list?

Reply:
Aren't the `gitter_url` and `biostars_url` enough?

Reply:
@nsoranzo Some people may prefer to send emails, for example to ask for help/information for new tutorial/topic if they are not sure. It may feel less "impressive" (maybe I am wrong)
@bgruening GTN mailing list there or for the `training@usegalaxy.eu` ?

Reply:
Its this one isn't it? http://galaxy-training-mailing-list-archive.35427.n7.nabble.com/